feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt

```diff
@@ -0,0 +1,169 @@
+package com.iteneum.profile.presentation
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.LocationOn
+import androidx.compose.material.icons.outlined.Mail
+import androidx.compose.material.icons.outlined.Phone
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.iteneum.designsystem.components.LpEditFloatingActionButton
+import com.iteneum.designsystem.components.LpOutlinedButton
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.profile.R
+
+@Composable
+fun ProfileView(
+
+) {
+    val sizes = LeasePertTheme.sizes
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(all = sizes.smallerSize)
+    ) {
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Box(
+                modifier = Modifier.padding(
+                    horizontal = sizes.nothingSize,
+                    vertical = sizes.midSmallSize
+                )
+            ) {
+                /*TODO here will be a nav component*/
+            }
+            Box(
+                modifier = Modifier
+                    .fillMaxWidth(0.75f)
+                    .padding(horizontal = sizes.minorSmallSize, vertical = sizes.minorRegularSize)
+            ) {
+                Text(
+                    text = stringResource(id = R.string.profile),
+                    fontSize = 22.sp,
+                    color = MaterialTheme.colorScheme.onPrimaryContainer
+                )
+            }
+            Column(
+                modifier = Modifier.padding(all = sizes.nothingSize),
+                horizontalAlignment = Alignment.End
+            ) {
+                LpEditFloatingActionButton(
+                    color = MaterialTheme.colorScheme.inversePrimary,
+                    colors = MaterialTheme.colorScheme.inversePrimary,
+                    elevation = FloatingActionButtonDefaults.elevation(sizes.nothingSize),
+                ) {}
+            }
+        }
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Column(modifier = Modifier.padding(horizontal = sizes.smallSize)) {
+                /*Image(
+                    painter = userPhoto,
+                    contentDescription = stringResource(R.string.user_photo),
+                    contentScale = ContentScale.Crop,
+                    modifier = Modifier
+                        .size(sizes.extraSize78)
+                        .clip(CircleShape)
+                )*/
+                /*TODO Here will be the image implementation*/
+            }
+            Column() {
+                Box(modifier = Modifier.padding(vertical = sizes.midSmallSize)) {
```

Comment:
Instead use two box, you could add the modifier to each texts

Reply:
Done! 👍